sourcegraph/cody
# Refactor eventlogger and use localstorage for server endpoint

* Refactor eventlogger so that the "shared" EventLogger.ts file doesn't contain any VSCode-specific logic; move all of that to the "vscode" event-logger.ts file instead
* Default to `localStorage.endpoint()` for server endpoint values for logging (`config.serverEndpoint` was deprecated as of 5.1 and now just returns an empty string in most cases)

The tsconfig updates were automatic when I compiled... If they shouldn't be included here, please let me know!

## Test plan

* run locally
* verify payload
* test that config updates are caught
